### PR TITLE
chore: update chart and dependencies versions

### PR DESCRIPTION
**Pull request by flemzord**
Update Chart.yaml to version 2.1.2 and operator dependency to 2.0.20. Update values.yaml with new versions for services in v2.0 and v2.1.

### CODE REVIEW

File: charts/regions/Chart.yaml [hunk 1]
```diff
@@ -20,6 +20,6 @@ dependencies:
     repository: "file://../agent"
     condition: agent.enabled
   - name: operator
-    version: v2.0.19
+    version: v2.0.20
```
Comment (Dav-14) [inline charts/regions/Chart.yaml:23]: Is v2.0.20 already published ?

Comment (flemzord) [reply charts/regions/Chart.yaml:23]: Yes, few day ago

**Comment by Dav-14** [comment]
The pre-commit does not pass

**Review by Dav-14 — APPROVED** [review]

File: charts/regions/README.md [hunk 2]
```diff
@@ -69,28 +69,28 @@ Formance Private Regions Helm Chart
...
+| versions.files."v2.1".gateway | string | `"v2.0.20"` |  |
+| versions.files."v2.1".ledger | string | `"v2.1.0"` |  |
+| versions.files."v2.1".operator-utils | string | `"v2.0.20"` |  |
+| versions.files."v2.1".orchestration | string | `"v2.0.20"` |  |
+| versions.files."v2.1".payments | string | `"v2.0.20"` |  |
+| versions.files."v2.1".reconciliation | string | `"v2.0.20"` |  |
+| versions.files."v2.1".search | string | `"v2.0.20"` |  |
+| versions.files."v2.1".stargate | string | `"v2.0.20"` |  |
+| versions.files."v2.1".wallets | string | `"v2.0.20"` |  |
+| versions.files."v2.1".webhooks | string | `"v2.0.20"` |  |
```
[the remark this answers is not in the document]
Comment (flemzord) [reply charts/regions/README.md:93]: Yes, Only the ledger version has changed

File: charts/regions/values.yaml [hunk 3]
```diff
@@ -79,26 +79,26 @@ versions:
...
+      search: v2.0.20
+      stargate: v2.0.20
+      auth: v2.0.20
+      wallets: v2.0.20
+      webhooks: v2.0.20
+      gateway: v2.0.20
+      payments: v2.0.20
+      orchestration: v2.0.20
+      reconciliation: v2.0.20
+      operator-utils: v2.0.20
```
Comment (Dav-14) [inline charts/regions/values.yaml:104]: Is this intended

Comment (flemzord) [reply charts/regions/values.yaml:104]: Yes, Only the ledger version has changed

**Pull request merged**